.github/workflows/build-kata-static-tarball-s390x.yaml

```diff
@@ -23,7 +23,7 @@ on:
 
 jobs:
   build-asset:
-    runs-on: s390x
+    runs-on: [s390x, build]
```

Comment:
Why "build"?

Reply:
I would like to divide our self-hosted runners into several categories. 😉